rust/impl/ca/ca/commands_aggregator.rs

```diff
@@ -6,11 +6,6 @@ pub( crate ) mod private
   use wtools::error::{ Result, BasicError };
   use wtools::string::split;
   use wtools::former::Former;
-  use log::
```

Comment:
Please, update code of tests. `Logger` is more powerful mechanism that `println`.